src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -629,7 +629,7 @@ class FileIdTracker {
 
   def getMaxFileId: Long = maxId
 
-  def getFileToIdMap: HashMap[key, Long] = fileToIdMap
+  def getFileToIdMap: Seq[(key, Long)] = fileToIdMap.toSeq
```

Comment:
not a getFileToId"Map" anymore?

Reply:
How about getFileToIdMapping, as technically "mapping" is just a concept and has nothing to do with the properties of the container?

Reply:
sgtm

Reply:
Done